build.sh

```diff
@@ -48,10 +48,15 @@ EOF
     echo "FC                     Path to Fortran compiler    (default is ${FC})"
     echo "BML_OPENMP             {yes,no}                    (default is ${BML_OPENMP})"
     echo "BML_MPI                {yes,no}                    (default is ${BML_MPI})"
+    echo "BML_MPIEXEC_EXECUTABLE Command to prepend MPI tests (default is ${BML_MPIEXEC_EXECUTABLE})"
+    echo "BML_MPIEXEC_NUMPROCS_FLAG Flags to specify number of MPI tasks (default is ${BML_MPIEXEC_NUMPROCS_FLAG})"
```

Comment:
Would it make sense to fold this option into the `BML_MPIEXEC_PREFLAGS` option as well?

Reply:
I am following cmake here: these are the variables cmake defines and uses based on MPI package found (without BML_ in front)

Reply:
Ah, I hadn't realized that. I agree with you to follow CMake's naming convention.